introduction/es/06-conclusion.asciidoc

```diff
@@ -0,0 +1,14 @@
+== Conclusiones
+
+En este módulo hemos visto una introducción a InnerSource.
+InnerSource aplica las buenas prácticas de desarrollo de los proyectos de software libre en desarrollos internos.
+Ofrece una opción adicional a los consumidores internos cuando los equipos que producen software no son capaces de llevar a cabo una nueva funcionalidad requerida.
+Para llevar a cabo InnerSource de forma satisfactoria, se ha de involucrar al https://innersourcecommons.org/learn/learning-path/product-owner/01[_product owner_] y a los https://innersourcecommons.org/learn/learning-path/trusted-committer/01[_trusted committers_] del *equipo anfitrión* así como a https://innersourcecommons.org/learn/learning-path/contributor/01[_contribuidores_] del *equipo invitado*
```

Comment:
On [03-how-it-works.asciidoc](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/introduction/es/03-how-works.asciidoc) the personas (PO, TC and Contribuidor) are capitalized. Should we align here or was it just for the (first) introduction of the concepts?

Reply:
Hey @fioddor , thanks for the comments. I've followed the same style as I found in the English version. I think it makes sense, at least in Spanish, to keep this as it is as the concepts were introduced back in that article. In other articles, this is not capitalized neither (https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/introduction/es/04-benefits.asciidoc).